cli-plugins/insight/query-somb-reports.hbs.md

```diff
@@ -0,0 +1,140 @@
+# Query Software Bill of Materials Reports
+
+This topic tells you how to query the database to retrieve a Software Bill 
+of Materials (commonly known as SBoM) report for a particular image. The information returned is similar
+to the SBoM returned by [querying for packages and CVE's a specific image contains](query-data.hbs.md#a-idexample1aexample-1-what-packages-and-cves-does-a-specific-image-contain)
```

Comment:
`CVE's` -> `CVEs` 

I'm not sure that it should have `'s` since I don't think you mean for it to be a possessive s.